code/modules/reagents/chemistry/reagents/drinks.dm

```diff
@@ -171,10 +177,11 @@
 	taste_message = "banana juice"
 
 /datum/reagent/consumable/drink/banana/on_mob_life(mob/living/M)
+	var/update_flags = STATUS_UPDATE_NONE
 	if((ishuman(M) && M.job in list("Clown") ) || issmall(M))
-		M.adjustBruteLoss(-1)
-		M.adjustFireLoss(-1)
-	..()
+		update_flags |= M.adjustBruteLoss(-1, FALSE)
+		update_flags |= M.adjustFireLoss(-1, FALSE)
```

Comment:
I do wonder what would be cheaper. doing this with the constant flags, or just doing something like `.=1`when damage occurs.

As an example here, would `|=` be necessary in both cases, as just one will be enough to trigger an update.

Thoughts?

Reply:
Here's an example:

```
				need_mob_update += R.on_mob_life(C)

	if(C && need_mob_update) //some of the metabolized reagents had effects on the mob that requires some updates.
		C.updatehealth()
		C.update_canmove()
		C.update_stamina()
```

```
/datum/reagent/medicine/morphine/on_mob_life(mob/living/carbon/M)
	switch(current_cycle)
		if(11)
			to_chat(M, "<span class='warning'>You start to feel tired...</span>" )
		if(12 to 24)
			M.drowsyness += 1
		if(24 to INFINITY)
			M.Sleeping(40, 0)
			. = 1
	..()
```

Reply:
A binary |= is a single CPU op. It's not like a list |=, which would be far more intensive.